### PR TITLE
feat(Popover): use theme.colors.secondary for outline color

### PR DESCRIPTION
**Pull request by saghdaey**
### Overview

Popover: use theme.colors.secondary for outline color
gamut-icons: add vertical menu icon

for file navigator sandbox work - i needed a white outline for the popover
and a vertical menu

Screen Shot 2021-05-07 at 6 42 54 PM

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: EGG-1134
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: packages/gamut-icons/src/svg/mini/mini-interface-setting-menu-vertical-icon.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<title>interface-setting-menu-vertical</title>
```
Comment (saghdaey) [inline packages/gamut-icons/src/svg/mini/mini-interface-setting-menu-vertical-icon.svg:2]: title is very long and causes overlap in storybook. see bottom left:
Screen Shot 2021-05-17 at 9 37 44 AM

File: packages/styleguide/stories/Molecules/Popover/index.stories.mdx [hunk 2]
```diff
@@ -54,6 +56,20 @@ Using the outline prop, you can add an outline to the popover.
...
+  <Story name="Dark" args={{ mode: 'dark' }}>
+    {(args) => {
+      return (
+        <ColorMode mode="dark">
+          <PopoverExample {...args} outline />
+        </ColorMode>
+      );
+    }}
+  </Story>
+</Canvas>
```
Comment (saghdaey) [inline packages/styleguide/stories/Molecules/Popover/index.stories.mdx:71]: do i need this? happy to delete; the background color is still white so its not very visible what i'm doing. we can update once we have the dark mode implemented for popover that includes background color?

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.12.3-alpha.deb0d5.0<br/>@codecademy/gamut-kit@0.3.103-alpha.deb0d5.0<br/>@codecademy/gamut-labs@17.1.1-alpha.deb0d5.0<br/>@codecademy/gamut@32.18.8-alpha.deb0d5.0<br/>@codecademy/styleguide@38.1.1-alpha.deb0d5.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60a2759f6a7f84120155e7e9--gamut-preview.netlify.app>https://60a2759f6a7f84120155e7e9--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60a2759f6a7f84120155e7e9>Deploy Logs</a></p>

File: packages/gamut-icons/src/svg/mini/mini-interface-setting-menu-vertical-icon.svg (hunk 1 above)
Comment (saghdaey) [reply packages/gamut-icons/src/svg/mini/mini-interface-setting-menu-vertical-icon.svg:2]: gonna rename..lol

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.12.3-alpha.d7403b.0<br/>@codecademy/gamut-kit@0.3.103-alpha.d7403b.0<br/>@codecademy/gamut-labs@17.1.1-alpha.d7403b.0<br/>@codecademy/gamut@32.18.8-alpha.d7403b.0<br/>@codecademy/styleguide@38.1.1-alpha.d7403b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60a295a355da890d7d0f9154--gamut-preview.netlify.app>https://60a295a355da890d7d0f9154--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60a295a355da890d7d0f9154>Deploy Logs</a></p>

File: packages/styleguide/stories/Molecules/Popover/index.stories.mdx (hunk 2 above)
Comment (JoshuaKGoldberg) [reply packages/styleguide/stories/Molecules/Popover/index.stories.mdx:71]: I'm ok with either way!

**Review by JoshuaKGoldberg — APPROVED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.12.3-alpha.3f3801.0<br/>@codecademy/gamut-kit@0.3.104-alpha.3f3801.0<br/>@codecademy/gamut-labs@17.1.2-alpha.3f3801.0<br/>@codecademy/gamut@32.18.9-alpha.3f3801.0<br/>@codecademy/styleguide@38.1.2-alpha.3f3801.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60a2a70c8110b10cd41b554b--gamut-preview.netlify.app>https://60a2a70c8110b10cd41b554b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60a2a70c8110b10cd41b554b>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60a51ea99210060357a770b8--gamut-preview.netlify.app>https://60a51ea99210060357a770b8--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60a51ea99210060357a770b8>Deploy Logs</a></p>

**Pull request merged**